cmd/up/organization/user/invite.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Upbound Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package user
+
+import (
+	"context"
+
+	"github.com/pterm/pterm"
+
+	"github.com/upbound/up-sdk-go/service/organizations"
+	"github.com/upbound/up/internal/upbound"
+	"github.com/upbound/up/internal/upterm"
+)
+
+// inviteCmd sends out an invitation to a user to join an organization.
+type inviteCmd struct {
+	OrgID      uint                                      `arg:"" required:"" help:"ID of the organization."`
+	Email      string                                    `arg:"" required:"" help:"Email address of the user to invite."`
+	Permission organizations.OrganizationPermissionGroup `short:"p" default:"member" help:"Role of the user to invite (owner or member)."`
```

Comment:
```suggestion
	Permission organizations.OrganizationPermissionGroup `short:"p" enum:"member,owner" default:"member" help:"Role of the user to invite (owner or member)."`
```
we can use this [tag](https://github.com/alecthomas/kong#supported-tags) to validate the input as well